core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -270,34 +273,105 @@ func (r *EvmRegistry) refreshActiveUpkeeps() error {
 	}
 	r.active.Reset(ids...)
 
-	return r.refreshLogTriggerUpkeeps(ids)
+	newUpkeeps, err := r.logEventProvider.RefreshActiveUpkeeps(ids...)
```

Comment:
just noticed that we were doing the filtering on upkeep type before calling this function. This will also work as current implementation of `r.logEventProvider.RefreshActiveUpkeeps` would just ignore those upkeeps, but maybe cleaner to do the filtering here?